adding-sites.mdx

```diff
@@ -19,7 +19,7 @@ information about which sites Sherlock will query usernames for, and how that qu
 carried out.
 
 Here is an illustrative entry for one site in the 
-[data.json](https://github.com/sherlock-project/sherlock/blob/master/data.json) file.  Following this, is a
+[data.json](https://github.com/sherlock-project/sherlock/blob/master/sherlock/resources/data.json) file.  Following this, is a
```

Comment:
Good catch.

That URL is soon to be invalid, though --- this one will remain active going forward

```suggestion
[data.json](https://github.com/sherlock-project/sherlock/blob/master/sherlock_project/resources/data.json) file.  Following this, is a
```

Reply:
Ahh oke, basically copied the one above. I've updated both to the new one :)